File: banner_manager/src/main/java/ru/nsu/fit/borzov/banner_manager/dto/AddBannerDto.java
package ru.nsu.fit.borzov.banner_manager.dto;

import lombok.Data;

import javax.validation.constraints.NotNull;

@Data
public class AddBannerDto {
    @NotNull
    private String name;
    @NotNull
    private Double price;
    @NotNull
    private String text;
    @NotNull
    private Long categoryId;
}

File: bm_front/src/Network/Dto/Banner/AddBannerDto.ts


export class AddBannerDto {
    name: string
    price: number
    categoryId: number
    text: string
    constructor(name: string = "", price: number = 0, categoryId: number = -1, text: string = "") {
        this.name = name;
        this.price = price;
        this.categoryId = categoryId;
        this.text = text;
    }
}